.storybook/main.ts

```diff
@@ -0,0 +1,27 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-mdx-gfm'],
+  features: {
+    buildStoriesJson: true,
+  },
+  previewHead: (head) => `
+    ${head}
+    <link rel="stylesheet" type="text/css" title="themeCss" href="styles/core-theme.css" />
+  `,
+  stories: [
+    '../packages/design-system/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
```

Comment:
We could change this matcher to `@([jt]sx)` and actually maybe we should go a step further and convert all of our `.jsx` stories to `.tsx` for consistency and just make this `...stories.tsx` with no matcher.

Reply:
Attempted to see if I could get away with changing the extensions as the stories are currently written, but ran into TS errors so this is probably more work than a quick rename.